### PR TITLE
Updated README.md

### LINKED ISSUE #18: Document about WebGL and Windows Build
### Description
- Create a section, `Run the Builds` in `README.md` 
- Mention the steps for both WebGL and Windows Build separately.

### PR DESCRIPTION
**Pull request by Karthick47v2**
### Details
Added Documentation about builds

### Checks
- [x] Closes #18

### CODE REVIEW

**Review by pranshi112 — CHANGES_REQUESTED** [review]
@Karthick47v2 There are several changes to make here:

1. Add the `Run the Builds` section at the end of `README.md`
2. For WebGL Build, also include an alternative that allows users to run the build on a web browser by starting a local server. Include the 2 methods like this:
**ALTERNATIVE 1:**
**ALTERNATIVE 2:**

**Comment by Karthick47v2** [comment]
@pranshi112 I have updated it, check it.

**Review by pranshi112 — CHANGES_REQUESTED** [review]
@Karthick47v2 Please include a method to start a local testing server using Python. Refer to the link  [here](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server).

Also, label the first method (that uses Servez) as **ALTERNATIVE 1**, and so on for other methods.

**Comment by Karthick47v2** [comment]
@pranshi112 Updated.

File: README.md [hunk 1]
```diff
@@ -37,3 +37,54 @@ A 2D animation that shows a menu with numerous food items that may be chosen fro
...
+2. While installing, make sure to check the __"Add Python 3.x to PATH"__ checkbox.
+3. Open your terminal and go to `Build\WebGL` or first go to `Build\WebGL` and type __"cmd"__ in address bar and enter.
+4. Type the following command.
+   ```
+   # For python 3
+   python -m http.server <portNum>
+   # For python 2
+   python -m SimpleHTTPServer <portNum>
+   ```
+   If your didn't put any `portNum` by default, this will run on port 8000.
```
Comment (pranshi112) [inline README.md]: I think it should be _you_ instead of _your_. Rest, everything looks good.

**Review by pranshi112 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Karthick47v2) [reply README.md]: Now its ok. Thanks for pointing this out.

**Review by pranshi112 — APPROVED** [review]

**Pull request merged**